lib/share_progress/button.rb

```diff
@@ -21,7 +35,8 @@ def destroy(id)
 
       def all(options={})
```

Comment:
I can get on board with this. It does give us control over the defaults. At some point down the road, I imagine the gem would have a config that would set those defaults too.